instauto/bot/bot.py

```diff
@@ -19,13 +19,14 @@ class Bot:
     _actions: List = []
     _post_cache: Dict[str, List[Dict]] = {}
 
-    def __init__(self, user_name: str, password: str, delay_between_action: float = 2.0) -> None:
+    def __init__(self, user_name: str, password: str, delay_between_action: float = 2.0, delay_tolerance_delta: float = 0.0) -> None:
         """Initiate a new `Bot` instance.
 
         Args:
             user_name: the username of the account
             password: the password of the account
             delay_between_action: the amount of seconds to wait between actions (each like, follow, etc. is an action)
+            delay_tolerance_delta: the amount of seconds to tolerance variing of the delay_between_action (to bring more randomness)
```

Comment:
"the amount of variance to add to the delay. Delay will be random number between (delay - variance) - (delay + variance).